hyperlink/_url.py

```diff
@@ -865,7 +849,7 @@ def absolute(self):
 
     def replace(self, scheme=_UNSET, host=_UNSET, path=_UNSET, query=_UNSET,
                 fragment=_UNSET, port=_UNSET, rooted=_UNSET, userinfo=_UNSET,
-                family=_UNSET, uses_netloc=_UNSET):
```

Comment:
If that's the case, then we need to rewrite this constructor to take `**kw` to avoid self-bombing.